controllers/operands/kubevirt_test.go

```diff
@@ -2524,6 +2524,121 @@ Version: 1.2.3`)
 			}))
 		})
 
+		Context("Tune KubeVirt rateLimiters", func() {
+
+			var hco *hcov1beta1.HyperConverged
+
+			BeforeEach(func() {
+				hco = commonTestUtils.NewHco()
+			})
+
+			It("Should be empty by default", func() {
+				kv, err := NewKubeVirt(nil, nil, hco)
+				Expect(err).ToNot(HaveOccurred())
+				Expect(hco.Spec.TuningPolicy).To(BeEmpty())
+				Expect(kv.Spec.Configuration.APIConfiguration).To(BeNil())
+				Expect(kv.Spec.Configuration.ControllerConfiguration).To(BeNil())
+				Expect(kv.Spec.Configuration.WebhookConfiguration).To(BeNil())
+				Expect(kv.Spec.Configuration.HandlerConfiguration).To(BeNil())
+
+			})
+
+			It("Should be empty if configmap is not present", func() {
+				hco.Spec.TuningPolicy = hcov1beta1.HyperConvergedTuningPolicyConfigMap
+				cl := commonTestUtils.InitClient([]runtime.Object{})
+
+				kv, err := NewKubeVirt(context.TODO(), cl, hco)
+
+				errT, ok := err.(*RateTuningError)
+
+				Expect(err).To(HaveOccurred())
+				Expect(ok).To(BeTrue())
+				Expect(errT).To(HaveOccurred())
```

Comment:
Not sure we need to check that if `ok` is `true`. If you want, you can check that `errT` is of the expected type. See here: https://onsi.github.io/gomega/#matcherrorexpected-interface

Reply:
Please, check if this is addressed properly.